ServantHDBC/src/Api/Photo.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Api.Photo (PhotoAPI, photoHandlers) where
+
+import Control.Monad.Trans.Except
+import Servant
+
+type PhotoAPI = "photos" :>
+                (Capture "path_segment_1" String :>
```

Comment:
so, I'm not clear what does naming the path segment really do? in this case `path_segment_1` and `path_segment_2`. Because the actual handler function anyways has to give it a variable name.